doc/gtd.norg

```diff
@@ -279,3 +279,6 @@ This is not in an AOF
       - Opening a particular project view
       - Opening a particular context view (or next actions)
       - Doing a weekly review
+
+$comment
```

Comment:
Do you want me to keep this?

Reply:
Actually, yes, we needed this in order to ensure the correct filetype and ensure highlighting etc.